### ISSUE TITLE
Script crashes trying to delete conversion.log

### ISSUE THREAD

**Issue by SpadarShut** [akosbalasko/yarle#239]
Trying to run the script fails with error:

```
Error: ENOENT: no such file or directory, unlink '/Users/***/Library/Application Support/yarle-evernote-to-md/conversion.log'
```

Need to check if the file exists before trying to remove it.

**Comment by akosbalasko** [comment]
Thanks for reporting this issue, i'll fix it soon

**Comment by akosbalasko** [comment]
Fixed in version 4.0.4
